go.mod

```diff
@@ -59,6 +61,7 @@ require (
 
 require (
 	github.com/go-openapi/strfmt v0.21.10
+	github.com/hashicorp/hcp-sdk-go v0.0.0-00010101000000-000000000000
```

Comment:
That's one original version spec, I assume it works since we replace, is the intent that we don't inadvertently commit this version spec once the `replace` is removed?

Anyway I believe this won't be part of the PR in its final state, so we can ignore this

Reply:
Yes, this won't be a part of final PR. Rightnow new protos are still not in the hcp-sdk-go that's why I am using the internal sdk.